src/app/component/comments/components/comments-list/comments-list.component.html

```diff
@@ -38,55 +38,65 @@
                 [value]="comment.text"
                 [formControl]="content"
                 class="edit-text-input"
-                maxlength="8000"></textarea>
-      <div class="save-cancel-wrapper">
-        <div class="save-cancel-btn">
-          <div class="save-btn">
+                maxlength="8000">
+      </textarea>
+    </div>
+    </div>
+  <div class="comments-elements">
+    <div class="btn-wrapper"
+         *ngIf="isLoggedIn && (comment.author.id === +userId) && !comment.isEdit; else editComment">
```

Comment:
https://stackoverflow.com/questions/17106681/parseint-vs-unary-plus-when-to-use-which
also, move complex logic to js file

Reply:
fixed.